Content.Shared/Psionics/Abilities/MassSleep/MassSleepPowerComponent.cs

```diff
@@ -1,18 +1,8 @@
-using Content.Shared.Actions;
-using Robust.Shared.Prototypes;
-using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
-
 namespace Content.Shared.Abilities.Psionics
 {
     [RegisterComponent]
     public sealed partial class MassSleepPowerComponent : Component
     {
         public float Radius = 1.25f;
```

Comment:
Is this a constant value? If so, why does it not have a `const`?

Reply:
I think you're correct here that it shouldn't be a variable that systems are allowed to modify, because more likely variables like this are the "Baseline" for something, such as this case where it's the "Unmodified" range at which Mass Sleep works. The intention is clearly that it should be a const defined by yml, and then never touched afterwards. That way you could make a variant MassSleepPower that has a different range.